selfservice/hook/web_hook.go

```diff
@@ -134,12 +135,13 @@ func (e *WebHook) ExecuteLoginPreHook(_ http.ResponseWriter, req *http.Request,
 func (e *WebHook) ExecuteLoginPostHook(_ http.ResponseWriter, req *http.Request, _ node.UiNodeGroup, flow *login.Flow, session *session.Session) error {
 	return otelx.WithSpan(req.Context(), "selfservice.hook.WebHook.ExecuteLoginPostHook", func(ctx context.Context) error {
 		return e.execute(ctx, &templateContext{
-			Flow:           flow,
-			RequestHeaders: req.Header,
-			RequestMethod:  req.Method,
-			RequestURL:     x.RequestURL(req).String(),
-			RequestCookies: cookies(req),
-			Identity:       session.Identity,
+			Flow:            flow,
```

Comment:
Isn't the internal context already available in the flow object itself? Or is it hidden in there?

Reply:
Yes, it's part of the flow object, but it does not serialise into json:

https://github.com/ory/kratos/blob/43e4eadce7fa6e66bf1f9c03136d141bffd3094f/selfservice/flow/login/flow.go#L86